Assets/Scripts/PlayerInteract.cs

```diff
@@ -4,27 +4,35 @@
 
 public class PlayerInteract : MonoBehaviour
 {
+    public GameObject interact_text;
```

Comment:
Are you sure you want to use '_' in variable names in C#?

Reply:
fixed